services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -393,11 +393,12 @@ func TestCreateUndeployApplicationVersionErrors(t *testing.T) {
 
 func TestDeployApplicationVersion(t *testing.T) {
 	tcs := []struct {
-		Name             string
-		Transformers     []Transformer
-		expectedError    string
-		expectedPath     string
-		expectedFileData []byte
+		Name                           string
+		Transformers                   []Transformer
+		expectedError                  string
+		expectedPath                   string
+		expectedFileData               []byte
+		expectedDeploymentMetadataPath string
```

Comment:
please also add a filed for `expectedDeploymentMetadataContent string` and test that the content is as expected.